pkg/app/piped/executor/kubernetes/kubernetes.go

```diff
@@ -277,7 +277,7 @@ func applyManifests(ctx context.Context, ag applierGetter, manifests []provider.
 		}
 
 		if err := applier.ApplyManifest(ctx, m); err != nil {
-			lp.Errorf("Failed to apply manifest: %s (%w)", m.Key.ReadableString(), err)
+			lp.Errorf("Failed to apply manifest: %s (%v)", m.Key.ReadableString(), err)
```

Comment:
```suggestion
			lp.Errorf("Failed to apply manifest: %s (%w)", m.Key.ReadableString(), err.Error())
```
I think this is what we're expecting here 👀 `%w` is for error format, the issue is instead of passing the err itself to persister, just pass the Error message of the error. Same for the other.